mgmtfn/k8splugin/types.go

```diff
@@ -0,0 +1,451 @@
+package k8splugin
+
+import "time"
+
+// ListMeta describes metadata that synthetic resources must have, including lists and
+// various status objects.
+type ListMeta struct {
```

Comment:
Shouldn't we be using these definitions from k8s instead of redefining it here?
Was there a problem importing k8s structure definitions?